src/ResourceManagement/Scheduler/Microsoft.Azure.Management.Scheduler/Generated/Models/BasicAuthentication.cs

```diff
@@ -33,7 +33,7 @@ public BasicAuthentication(HttpAuthenticationType? type = default(HttpAuthentica
         public string Username { get; set; }
```

Comment:
Here is the swagger change PR: https://github.com/Azure/azure-rest-api-specs/pull/863